torch_xla/csrc/tensor_util.h

```diff
@@ -146,8 +146,50 @@ inline bool tensor_has_dym_dim(at::Tensor t) {
   return !c10::asIntArrayRefSlowOpt(sym_sizes).has_value();
 }
 
+inline bool tensors_has_dym_dim(at::TensorList tensors) {
+  for (const at::Tensor t : tensors) {
```

Comment:
maybe `const at::Tensor& t` to avoid the copy here?

Reply:
Thanks. Fixed.